website/search_migration/migrate.py

```diff
@@ -71,6 +71,7 @@ def migrate(delete, index=None, app=None):
         delete_old(new_index)
 
     ctx.pop()
+    populate_institutions.main('prod')
```

Comment:
This is going to break staging, staging2, test, etc. Make sure this populates the proper institution for the environment it's on.